server/src/main/java/com/octopus/teamcity/opentelemetry/server/ProjectConfigurationSettingsController.java

```diff
@@ -133,8 +133,8 @@ public boolean validate(@NotNull ActionErrors errors) {
             if (!this.service.isPresent())
                 errors.addError("service", "Service must be set!");
             else {
-                if (!this.service.get().equals(OTELService.HONEYCOMB) && !this.service.get().equals(OTELService.CUSTOM))
-                    errors.addError("service", "Service must be set to either 'honeycomb.io' or 'custom'!");
+                if (!this.service.get().equals(OTELService.HONEYCOMB) && !this.service.get().equals(OTELService.CUSTOM) && !this.service.get().equals(OTELService.ZIPKIN))
```

Comment:
Just a suggestion, maybe its time to move these many `ifs` to different methods or using switches

Reply:
Yeah, completely agree.
I'm thinking of hanging some kind of `PropertyAdapter`s off the service enum - that would clean this up quite nicely.

Reply:
But... later... Not keen to do it as part of this PR.